selfdrive/car/hyundai/carstate.py

```diff
@@ -132,8 +132,8 @@ def update(self, cp):
 
     # There is a 2% error on Sorento GT due to slightly larger wheel diameter compared to poverty packs.
     #   Dash assumes about 4% which is excessive. Using 3% to be safe
-    if self.car_fingerprint == CAR.SORENTO:
-      self.v_wheel = self.v_wheel * 1.03 
+    if self.car_fingerprint == CAR.GENESIS:
+      self.v_wheel = self.v_wheel * 1.075 
```

Comment:
Have you checked this against GPS?
Your Dash could be reporting a false high speed (common for most manufacturers)
We want the speed to be accurate, not equal to the dash.